SQL/src/test/java/com/walmart/SQL/MySteppers.java

```diff
@@ -51,7 +52,8 @@ public void theClientPostsToStudents() throws IOException {
         HttpEntity<String> entity = new HttpEntity<String>(jsonBody,headers);
 
         restTemplate = new RestTemplate();
-        latestResponse = restTemplate.postForEntity("localhost:8080/students",entity,String.class);
+        ResponseEntity<String> postForEntity = restTemplate.postForEntity("http://localhost:8000/students",entity,String.class);
+        latestResponse = new ResponseResults(postForEntity);
```

Comment:
I guess you got confused by following two tutorials. If you use `postForEntity`, you'll get `ResponseEntity` object. Just converting that to your `ResponseResults` object